### ISSUE TITLE
google/apiclient : dev-master does not exist

### ISSUE THREAD

**Issue by Jolora** [pulkitjalan/google-apiclient#2]
Hi,

On running `composer require pulkitjalan/google-apiclient` I receive the following error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for pulkitjalan/google-apiclient 0.1.* -> satisfiable by pulkitjalan/google-apiclient[0.1.0].
    - pulkitjalan/google-apiclient 0.1.0 requires google/apiclient dev-master -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

**Comment by spirant** [comment]
I have the same problem. I have tried adding the "google/apiclient": "dev-master"t to the composer.json file and got a lot of text back!

````
$ composer update | clip
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove laravel/framework v5.0.31
    - Conclusion: don't install laravel/framework v5.0.31
    - Conclusion: don't install laravel/framework 5.0.30
    - Conclusion: don't install laravel/framework v5.0.29
    - Conclusion: don't install laravel/framework v5.0.28
    - Conclusion: don't install laravel/framework v5.0.27
    - Conclusion: don't install laravel/framework v5.0.26
    - Conclusion: don't install laravel/framework v5.0.25
    - Conclusion: don't install laravel/framework v5.0.24
    - Conclusion: don't install laravel/framework v5.0.23
    - Conclusion: don't install laravel/framework v5.0.22
    - Conclusion: don't install laravel/framework v5.0.21
    - Conclusion: don't install laravel/framework v5.0.20
    - Conclusion: don't install laravel/framework v5.0.19
    - Conclusion: don't install laravel/framework v5.0.18
    - Conclusion: don't install laravel/framework v5.0.17
    - Conclusion: don't install laravel/framework v5.0.16
    - Conclusion: don't install laravel/framework v5.0.15
    - Conclusion: don't install laravel/framework v5.0.14
    - Conclusion: don't install laravel/framework v5.0.13
    - Conclusion: don't install laravel/framework v5.0.12
    - Conclusion: don't install laravel/framework v5.0.11
    - Conclusion: don't install laravel/framework v5.0.10
    - Conclusion: don't install laravel/framework v5.0.9
    - Conclusion: don't install laravel/framework v5.0.8
    - Conclusion: don't install laravel/framework v5.0.7
    - Conclusion: don't install laravel/framework v5.0.6
    - Conclusion: don't install laravel/framework v5.0.5
    - Conclusion: don't install laravel/framework v5.0.4
    - Conclusion: don't install laravel/framework v5.0.3
    - Installation request for pulkitjalan/google-apiclient dev-master -> satisf
iable by pulkitjalan/google-apiclient[dev-master].
    - Conclusion: don't install laravel/framework v5.0.2
    - Conclusion: don't install laravel/framework v5.0.1
    - pulkitjalan/google-apiclient dev-master requires illuminate/support ~4 ->
satisfiable by illuminate/support[v4.0.0, v4.0.1, v4.0.10, v4.0.2, v4.0.3, v4.0.
4, v4.0.5, v4.0.6, v4.0.7, v4.0.8, v4.0.9, v4.1.0, v4.1.1, v4.1.10, v4.1.11, v4.
1.12, v4.1.13, v4.1.14, v4.1.15, v4.1.16, v4.1.17, v4.1.18, v4.1.19, v4.1.2, v4.
1.20, v4.1.21, v4.1.22, v4.1.23, v4.1.24, v4.1.25, v4.1.26, v4.1.27, v4.1.28, v4
.1.29, v4.1.3, v4.1.30, v4.1.4, v4.1.5, v4.1.6, v4.1.7, v4.1.8, v4.1.9, v4.2.1,
v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.
8, v4.2.9].
    - don't install illuminate/support v4.0.0|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.1|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.10|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.0.2|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.3|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.4|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.5|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.6|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.7|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.8|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.0.9|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.0|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.1|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.10|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.11|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.12|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.13|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.14|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.15|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.16|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.17|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.18|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.19|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.2|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.20|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.21|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.22|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.23|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.24|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.25|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.26|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.27|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.28|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.29|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.3|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.30|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.1.4|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.5|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.6|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.7|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.8|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.1.9|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.1|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.12|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.2.16|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.2.17|don't install laravel/framework v
5.0.0
    - don't install illuminate/support v4.2.2|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.3|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.4|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.5|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.6|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.7|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.8|don't install laravel/framework v5
.0.0
    - don't install illuminate/support v4.2.9|don't install laravel/framework v5
.0.0
    - Installation request for laravel/framework 5.0.* -> satisfiable by laravel
/framework[5.0.30, v5.0.0, v5.0.1, v5.0.10, v5.0.11, v5.0.12, v5.0.13, v5.0.14,
v5.0.15, v5.0.16, v5.0.17, v5.0.18, v5.0.19, v5.0.2, v5.0.20, v5.0.21, v5.0.22,
v5.0.23, v5.0.24, v5.0.25, v5.0.26, v5.0.27, v5.0.28, v5.0.29, v5.0.3, v5.0.31,
v5.0.4, v5.0.5, v5.0.6, v5.0.7, v5.0.8, v5.0.9].
````

**Comment by mikevrind** [comment]
I guess this is caused by using L5 with this package. I created a new PR with  "illuminate/support": "~5"

**Comment by pulkitjalan** [comment]
hmm, yea but the service provider needs to be modified to work with Laravel 5 also. Ill have a look.

**Comment by markmooibroek** [comment]
Any update on this?

**Comment by pulkitjalan** [comment]
Sorry for the delay, released a new version `0.2.0` with the fix.
